system/jlib/jlib.hpp

```diff
@@ -204,6 +204,8 @@ class PointerArrayOf : public PointerArray
     inline bool zap(TYPE * x)                   { return PointerArray::zap(x); }
 };
 
+enum SSLCfg { SSLNone = 0, SSLBoth, SSLOnly };
```

Comment:
It makes it look like there's a reason, like you're depending on it being 0.
Which in fact you are with code like if (useSSL), which make it sound like it's a boolean not an enumarated type.